src/libraries/System.Runtime/tests/System/CharTests.cs

```diff
@@ -207,6 +207,43 @@ public static void IsAscii_Char()
             Assert.False(char.IsAscii('\x0080'));
             Assert.False(char.IsAscii(char.MaxValue));
         }
+        [Fact]
+        public static void IsAsciiDigit_Char() =>
+            IsAsciiLetterOrDigit(UnicodeCategory.DecimalDigitNumber);
+
+        [Fact]
+        public static void IsAsciiLetter_Char() =>
+            IsAsciiLetterOrDigit(
+                UnicodeCategory.UppercaseLetter,
+                UnicodeCategory.LowercaseLetter);
+
+        [Fact]
+        public static void IsAsciiLetterLower_Char() =>
+            IsAsciiLetterOrDigit(UnicodeCategory.LowercaseLetter);
+
+        [Fact]
+        public static void IsAsciiLetterUpper_Char() =>
+            IsAsciiLetterOrDigit(UnicodeCategory.UppercaseLetter);
```

Comment:
These tests are confusing me a bit.

None of them seem to be testing the method their title suggests, they're all doing IsAsciiLetterOrDigit.

Further, IsAsciiLetterOrDigit isn't actually testing that new method, it's just calling IsAscii and IsLetter (seemingly missing digits)

Reply:
Uhh... they're confusing to me, too.  Fixed.